.gitignore

```diff
@@ -83,3 +83,5 @@ silk/tests/test_unit_LPC_inv_pred_gain
 src/Debug
 src/Release
 src/x64
+build
```

Comment:
Use slashes as conventional in git when specifying directories in paths, e.g. `/build` or `build/`, otherwise it could match files.

Reply:
/*[Bb]uild*/

Ignores al commonl variants: `build`, `cmake-build`, `CMakeBuild` etc.

Reply:
Updated based on suggestions